geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9990,6 +10011,43 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  PartitionRegionConfig updatePRNodeInformation() {
+    if (cache.getDistributedSystem().getProperties().get(ENABLE_CLUSTER_CONFIGURATION) != "true") {
+      return null;
+    }
+    CacheLoader cacheLoader = basicGetLoader();
+    CacheWriter cacheWriter = basicGetWriter();
```

Comment:
Should cacheLoader and cacheWriter be read under lock, in case someone changes these fields multiple times?

Reply:
Done